app/src/datasets/chorus_dt/places_resolution.py

```diff
@@ -53,6 +54,29 @@ def compute_distance_between_points(lon0, lat0, lon1, lat1):
 
     return c * r
 
+def calc_CO2(trips:pandas.DataFrame,
+             CO2_frame:pandas.DataFrame):
+    
+    short_plane = (trips['prestation'] == 'A') & (trips['distance'] <= 1000)
+    long_plane = (trips['prestation'] == 'A') & (trips['distance'] > 1000)
+    train = (trips['prestation'] == 'T')
+    commun = (trips['prestation'] == 'TC')
+    
+    CO2_short_plane = CO2_frame.loc[(CO2_frame['Prestation'] == 'A') &
+                                    (CO2_frame['filtre'] == '0-1000 km'), 'Total poste non décomposé'].iloc[0]
+    CO2_long_plane = CO2_frame.loc[(CO2_frame['Prestation'] == 'A') &
+                                    (CO2_frame['filtre'] == '> 1000 km'), 'Total poste non décomposé'].iloc[0]
+    CO2_TGV = CO2_frame.loc[(CO2_frame['Prestation'] == 'T') &
+                                   (CO2_frame['filtre'] == 'TGV'), 'Total poste non décomposé'].iloc[0]
+    CO2_TC = CO2_frame.loc[(CO2_frame['Prestation'] == 'TC'), 'Total poste non décomposé'].iloc[0]
+    
+    
+    trips['kgCO2e/passager.km'] = short_plane.astype(int)*CO2_short_plane + long_plane.astype(int)*CO2_long_plane + train.astype(int)*CO2_TGV + commun.astype(int)*CO2_TC
```

Comment:
Ok seems legit :)

Maybe it would be more readable to have something like :
```
trips.loc[short_plane][''kgCO2e/passager.km'] = CO2_short_plane
trips.loc[long_plane][''kgCO2e/passager.km'] = CO2_long_plane
```